caos.ansible_roles/roles/assert_host_status_event/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: list host_status change events
+  delegate_to: 127.0.0.1
+
+  import_role:
+    name: caos.ansible_roles.nrql_api_request
```

Comment:
when referring to roles from caos-ansible-roles we are not using the prefix (`caos.ansible_roles`) and we are using `include_role`.

For the prefix I'm not sure if there is any good practice (and my memory says that with the prefix I had some trouble but it could be wrong) but I think we should use the "same convention" (always with it or without it, not strong opinion)

For using import or include the same. I don't have strong opinion (more info here https://www.ansiblejunky.com/blog/ansible-101-include-vs-import/) but we should use the same.

wdyt?

Reply:
I agree with both and I have updated it in the last commit.